### ISSUE TITLE
Resoning behind the epochs chosen for megaage_asian

### ISSUE THREAD

**Issue by codezart** [redcping/Delta_Age_AdaIN#3]
The model uses 200 epochs as mentioned in the config file. But when we tested it with our training the model seemed to be overfitting after 30 epochs.

Can you please tell why you chose 200 epochs?

**Comment by codezart** [comment]
@redcping

**Comment by redcping** [comment]
Thank you for your attention. First, make sure that you have configured random crop and rotation by increasing the corresponding probabilities to prevent early overfitting. Additionally, during data processing, we expand the background surrounding the facial region by 1.2 times to ensure the accuracy of random crop.

**Comment by kerlenrikun** [comment]
数据集在哪里

**Comment by redcping** [comment]
> 数据集在哪里

http://mmlab.ie.cuhk.edu.hk/projects/MegaAge/
